src/Components/Landing/Login.js

```diff
@@ -47,24 +47,28 @@ const Login = props => {
       </input>
       <button onClick={e => submit()}>Login</button>
       <button>I need access</button> */}
-      <input className="btn-1 landing-page-adjust" type="button"
+      <input className="btn-1" type="button"
         value="Login"
-        onClick={togglePopup} />
-      {isOpen && <Popup content={<>
+        onClick={(e) => togglePopup()}
+      />
+
+
+      {/* THIS IS THE POPUP THAT APPEARS WHEN "LOGIN" IS CLICKED ON THE LANDING PAGE */}
+      {isOpen ? <Popup content={<>
         <div className="popup-header">
           {/* <img className="logo" src={logo} ></img> */}
-          <h2>Lemon Prop Mgmt</h2>
+          <h2>Lemon Prop Login</h2>
```

Comment:
We should remove the space between Lemon and Prop here

Reply:
great catch!

Reply:
Are you good to do that when we approve the pull request?

Reply:
I just merged. If you're going to be working on it today, I'd say you go ahead and do it. If it's still there when I get back to it next, I'll make sure it gets taken care of for sure.

Reply:
you rock!